app/config/ViewConfig.scala

```diff
@@ -26,26 +26,26 @@ final case class ViewConfig(
     signInPath:     String,
     //    signOut:              String,
     //    frontendBaseUrl:      String,
-    frontendBaseUrl:               String,
-    accessibilityStatementBaseUrl: String,
-    accessibilityStatementPath:    String,
-    timeoutDialogTimeout:          Int,
-    timeoutDialogCountdown:        Int
+    frontendBaseUrl: String,
+    //    accessibilityStatementBaseUrl: String,
```

Comment:
if we're not using these things we should be able to just delete them. There's a few commented out lines here which we could clean up